EventsExpress.Db/Migrations/20201104143929_AddOccurenceEventTable.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace EventsExpress.Db.Migrations
+{
+    public partial class AddOccurenceEventTable : Migration
```

Comment:
Please, drop all migrations that you generated and generate a single one. No need to store all history of modifications.